### PR TITLE
python3Packages.sev-snp-measure: init at 0.0.9

### PR DESCRIPTION
**Pull request by msanft**
## Description of changes

Add the [sev-snp-measure](https://github.com/virtee/sev-snp-measure) Python package to create measurements for SEV-SNP CVMs. 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by msanft** [comment]
Result of `nixpkgs-review pr 303158` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python311Packages.sev-snp-measure</li>
    <li>python311Packages.sev-snp-measure.dist</li>
    <li>python312Packages.sev-snp-measure</li>
    <li>python312Packages.sev-snp-measure.dist</li>
  </ul>
</details>

**Comment by cole-h** [comment]
You'll want to rebase on the latest `master` -- there was an issue that broke ofborg's ability to evaluate Nixpkgs that has since been resolved. Sorry for the inconvenience!

File: pkgs/development/python-modules/sev-snp-measure/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  pythonImportsCheck = [ "sevsnpmeasure" ];
+
+  meta = {
+    description = "Calculate AMD SEV/SEV-ES/SEV-SNP measurement for confidential computing";
+    homepage = "https://github.com/virtee/sev-snp-measure";
+    changelog = "https://github.com/virtee/sev-snp-measure/releases/tag/v${version}";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ msanft ];
+    mainProgram = "sev-snp-measure";
+    platforms = lib.platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sev-snp-measure/default.nix:45]:
```suggestion
```
buildPythonPackage  defaults to unix which is probably more correct here

File: pkgs/development/python-modules/sev-snp-measure/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  pname = "sev-snp-measure";
+  version = "0.0.9";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    hash = "sha256-efW4DMple26S3Jizc7yAvdPjVivyMJq4fEdkuToamGc=";
+    owner = "virtee";
+    repo = "sev-snp-measure";
+    rev = "v${version}";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sev-snp-measure/default.nix:19]:
```suggestion
    owner = "virtee";
    repo = "sev-snp-measure";
    rev = "v${version}";
    hash = "sha256-efW4DMple26S3Jizc7yAvdPjVivyMJq4fEdkuToamGc=";
```

File: pkgs/development/python-modules/sev-snp-measure/default.nix [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  pythonImportsCheck = [ "sevsnpmeasure" ];
+
+  meta = {
+    description = "Calculate AMD SEV/SEV-ES/SEV-SNP measurement for confidential computing";
+    homepage = "https://github.com/virtee/sev-snp-measure";
+    changelog = "https://github.com/virtee/sev-snp-measure/releases/tag/v${version}";
+    license = lib.licenses.asl20;
+    maintainers = with lib.maintainers; [ msanft ];
+    mainProgram = "sev-snp-measure";
+    platforms = lib.platforms.unix;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sev-snp-measure/default.nix:45]:
```suggestion
```
buildPythonPackage already sets this and we should not repeat the same value

**Comment by SuperSandro2000** [comment]
@ofborg build python3Packages.sev-snp-measure

**Pull request merged**